pkgs/development/python-modules/python-daemon/default.nix

```diff
@@ -20,6 +20,10 @@ buildPythonPackage rec {
   propagatedBuildInputs = [ docutils lockfile ];
 
   checkInputs = [ pytest mock testscenarios ];
+
+  # Test suite is not compatible with pytest > 5.3.5
+  doCheck = false;
```

Comment:
```
===== 39 failed, 204 passed, 2 deselected, 2 warnings, 32 errors in 3.43s ======
```
Possible we could do better here with patching and disabling, maybe?

https://hydra.nixos.org/build/120804609